checkstyle/suppressions.xml

```diff
@@ -181,7 +181,7 @@
               files="StreamsPartitionAssignor.java"/>
 
     <suppress checks="NPathComplexity"
-              files="(ProcessorStateManager|InternalTopologyBuilder|StreamsPartitionAssignor|StreamThread|TaskManager|AssignorConfiguration).java"/>
+              files="(AssignorConfiguration|InternalTopologyBuilder|KafkaStreams|ProcessorStateManager|StreamsPartitionAssignor|StreamThread|TaskManager).java"/>
```

Comment:
Do we really need to add this exception? How much work would it be to reduce the complexity of `KafkaStreams`?

Reply:
NPathComplexity is a tough one to work around. We'd wind up having to move some blocks of logic to separate helper classes.